nemo/collections/nlp/models/duplex_text_normalization/duplex_decoder.py

```diff
@@ -50,6 +51,22 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
         # Language
         self.lang = cfg.get('lang', None)
 
+        # Covering Grammars
+        self.cg_normalizer = None  # Default
+        # We only support integrating with English TN covering grammars at the moment
+        self.use_cg = cfg.get('use_cg', False) and self.lang == constants.ENGLISH
+        if self.use_cg:
+            self.setup_cg(cfg.get('neural_confidence_threshold', 0.99))
+
+    # Setup covering grammars
+    def setup_cgs(self, neural_confidence_threshold):
+        self.use_cg = True
```

Comment:
this should be always true before setup_cgs is called? so is this redundant?

Reply:
add doc strings

Reply:
Added the doc strings.